lib/src/util/io.dart

```diff
@@ -142,13 +146,31 @@ Future<T> getUnusedPort<T>(FutureOr<T> tryPort(int port)) async {
 /// This has a built-in race condition: another process may bind this port at
 /// any time after this call has returned. If at all possible, callers should
 /// use [getUnusedPort] instead.
-Future<int> getUnsafeUnusedPort() async {
-  var socket = await RawServerSocket.bind(InternetAddress.LOOPBACK_IP_V4, 0);
+Future<int> getUnsafeUnusedPort({bool IPv6: false}) async {
```

Comment:
Tricky. It's "IPv6" in prose. According to the guidelines, the acronym would get lowercased, but it's not clear what that means for the "v". I think either `ipv6` or `ipV6` would work.